cg/services/order_validation_service/models/existing_sample.py

```diff
@@ -0,0 +1,14 @@
+from pydantic import BaseModel, Field
+
+from cg.models.orders.sample_base import NAME_PATTERN, StatusEnum
+
+
+class ExistingSample(BaseModel):
+    father: str | None = Field(None, pattern=NAME_PATTERN)
+    internal_id: str
+    mother: str | None = Field(None, pattern=NAME_PATTERN)
+    status: StatusEnum
```

Comment:
Same here, I don't think it makes sense to have the status as a required field when providing an existing sample.
But disregard for now unless you agree.

Reply:
The status is set on the `case_sample` entries which are new even for existing cases (provided they are within a new case). So I don't think it would work to set it as optional given that we don't know where to fetch it from.

Reply:
Sounds like the status does not belong on the case sample model but rather on the sample model then. Definitely out of scope here, but that is some weird data modelling